src/main/webapp/app/overview/exercise-details/request-feedback-button/request-feedback-button.component.ts

```
@@ -0,0 +1,124 @@
+        if (!this.assureConditionsSatisfied()) {
+            return;
+        }
+
+        if (this.exercise().type === ExerciseType.PROGRAMMING) {
+            const confirmLockRepository = this.translateService.instant('artemisApp.exercise.lockRepositoryWarning');
+            if (!window.confirm(confirmLockRepository)) {
+                return;
+            }
+        }
+
+        this.courseExerciseService.requestFeedback(this.exercise().id!).subscribe({
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)